demos/database table/workflow_database_tables.sql

```diff
@@ -0,0 +1,46 @@
+SET sql_mode = STRICT_ALL_TABLES;
+
+DROP TABLE IF EXISTS Workflow;

```

Comment:
Nit picky but it seems some tables are plural and some aren't. Should make them follow a logical pattern.